api/routers/edr.py

```diff
@@ -151,9 +158,8 @@ async def get_data_location_id(
     #  This is just a quick and dirty demo
     range = get_datetime_range(datetime)
     if parameter_name:
-        parameter_name = parameter_name.split(",")
-        parameter_name = list(map(lambda x: x.strip(), parameter_name))
-    # parameter_name = verify_parameter_names(parameter_name) # should the api verify that the parameter name is valid?
+        parameter_name = split_and_strip(parameter_name)
+        await verify_parameter_names(parameter_name)
     request = dstore.GetObsRequest(
         filter=dict(
             parameter_name=dstore.Strings(values=parameter_name),
```

Comment:
Should set `parameter_name` to None in the dict if no parameters are given.
We now pass in an empty string, which works (as that matches anything), but it is not really correct.

Reply:
Will make separate issue for this.